### PR TITLE
modified build script to allow for building fakeclients (internal use only)

### PR DESCRIPTION
**Pull request by kevinimprobable**
#### Description
Changes to the build scripts to add a script to allow building fakeclients.
#### Tests
I've built numerous fakeclients with this script and run deployments with these workers.
#### Primary reviewers
@girayimprobable @Vatyx

### CODE REVIEW

File: SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs [hunk 1]
```diff
@@ -21,6 +21,25 @@ shift 1
 chmod +x $SCRIPT
 gosu $NEW_USER ""${{SCRIPT}}"" ""$@"" 2> >(grep -v xdg-user-dir >&2)";
 
+
+        // This is for internal use only. We do not support Linux clients.
+        private const string FakeClientWorkerShellScript =
+			@"#!/bin/bash
```
Comment (Vatyx) [inline SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs]:
```suggestion
@"#!/bin/bash
```

Comment (kevinimprobable) [reply SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs]: The other worker shell script also had this line indented like this (at the top of the file). Should I un-indent that one as well?

Comment (Vatyx) [reply SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs]: eeeeeeehhhhhhhhhhhhhHHHHHHHHHHHHHHHHHHHHH probably

Comment (kevinimprobable) [reply SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs]: this is now done

**Pull request merged**